internal/restarter/sidecars.go

```diff
@@ -119,7 +119,7 @@ func (s *SidecarsRestarter) Restart(ctx context.Context, istioCR *v1alpha2.Istio
 		if warningsCount-len(pods) > 0 {
 			warningMessage += fmt.Sprintf(" and %d additional workload(s)", warningsCount-len(pods))
 		}
-		warningErr := described_errors.NewDescribedError(errors.New("Istio controller could not restart one or more istio-injected pods."), "Not all pods with Istio injection could be restarted. Please take a look at kyma-system/istio-controller-manager logs to see more information about the warning").SetWarning()
+		warningErr := described_errors.NewDescribedError(errors.New("Istio Controller could not restart one or more Istio-injected Pods."), "Not all the Pods with Istio sidecar injection could be restarted. To learn more about the warning, see kyma-system/istio-controller-manager logs.").SetWarning()
```

Comment:
> Not all the Pods with Istio sidecar injection could be restarted

This still sounds a bit wrong to me. Maybe something like `Some Pods with Istio sidecar injection failed to restart` would be better?